themes/_core/js/checkout-payment.js

```diff
@@ -146,6 +155,14 @@ class Payment {
       return;
     }
 
+    $.post(window.location.href, {
```

Comment:
We shouldn't rerun the `window.location.href`, more if it concerns the legacy page.
Legacy pages are handle depending on `_GET` parameters, meaning if you have a module injecting his own parameters, you will maybe execute code that couldn't be execute.
Please use routes instead.

Reply:
ping @NeOMakinG help pls

Reply:
`window.prestashop.urls.pages.order`

Reply:
I believe @Matt75 provided a nice tip to be used ;)